server/rpc_test.go

```diff
@@ -650,6 +650,8 @@ func TestValidatePrice(t *testing.T) {
 	err := validatePrice(s)
 	assert.Nil(err)
 
+	defer BroadcastCfg.SetMaxPrice(nil)
```

Comment:
This was needed because this package var was not being reset which affected other tests that depend on it.